packages/web-app-files/src/views/spaces/Projects.vue

```diff
@@ -6,20 +6,21 @@
       ref="createNewSpaceButton"
       key="new-space-menu-btn-enabled"
       :aria-label="$gettext('Create a new space')"
-      variation="primary"
-      appearance="filled"
-      class="oc-mb-l"
+      variation="inverse"
+      class="oc-mb-l oc-background-primary-gradient"
       data-testid="spaces-list-create-space-btn"
       @click="showCreateSpaceModal"
     >
       <oc-icon name="add" />
       <translate>Create Space</translate>
     </oc-button>
-    <span class="oc-display-block oc-mb-l">
-      <span v-text="$gettext('Access all project related files in one place.')" />
-      <a href="#" v-text="$gettext('Learn more about spaces.')" />
+    <span class="oc-display-block oc-mb-xl">
+      <span
+        v-text="$gettext('Store your project related files in Spaces for seamless collaboration.')"
+      />
+      <!-- <a href="#" v-text="$gettext('Learn more about spaces.')" /> -->
     </span>
-    <hr class="oc-mb-s" />
+    <hr class="oc-mb-l oc-border-b" />
```

Comment:
Feels a bit weird to give a bottom border to an `<hr>` 🤔  I'd prefer if you'd restyle the top border to be thicker instead of having a bottom and a top border applied at the same time.

By the way, accessibility wise, the `<hr>` tag will be announced via screen readers as horizontal divider. Was that intended? If not you'd need to give the block above a bottom border instead of adding the `<hr>`.